perception_dataset/deepen/deepen_to_t4_converter.py

```diff
@@ -66,34 +67,54 @@ def __init__(
         else:
             self._surface_categories = []
 
-    def convert(self):
+    def _format_annotations(self) -> Dict[str, Dict[str, Any]]:
+        """Format annotations from input_anno_file."""
         camera_index: Dict[str, int] = self._description["camera_index"]
         if self._label_info is None:
             deepen_annotations = DeepenAnnotation.from_file(self._input_anno_file)
-        else:
-            if self._label_info.label_type == LabelType.SEGMENTATION_2D:
-                deepen_annotations = (
-                    DeepenSegmentationPolygon2D.from_file(
-                        ann_file=self._input_anno_file,
-                        data_root=self._input_base,
-                        camera2index=camera_index,
-                        dataset_corresponding=self._t4data_name_to_deepen_dataset_id,
-                    )
-                    if self._label_info.label_format == LabelFormat.POLYGON
-                    else DeepenSegmentationPainting2D.from_file(
-                        ann_file=self._input_anno_file,
-                        data_root=self._input_base,
-                        camera2index=camera_index,
-                        dataset_corresponding=self._t4data_name_to_deepen_dataset_id,
-                    )
+        elif self._label_info.label_type == LabelType.SEGMENTATION_2D:
+            deepen_annotations = (
+                DeepenSegmentationPolygon2D.from_file(
+                    ann_file=self._input_anno_file,
+                    data_root=self._input_base,
+                    camera2index=camera_index,
+                    dataset_corresponding=self._t4data_name_to_deepen_dataset_id,
+                )
+                if self._label_info.label_format == LabelFormat.POLYGON
+                else DeepenSegmentationPainting2D.from_file(
+                    ann_file=self._input_anno_file,
+                    data_root=self._input_base,
+                    camera2index=camera_index,
+                    dataset_corresponding=self._t4data_name_to_deepen_dataset_id,
+                )
+            )
+        elif self._label_info.label_type == LabelType.POINT_3D:
+            if self._label_info.label_type == LabelType.POINT_3D:
```

Comment:
Did you intend to compare `label_format` at this line?
```suggestion
            if self._label_info.label_format == LabelFormat.PAINT_3D:
```

Reply:
Good catch, I have fixed that 
Would you mind taking a look again? Thanks